Gemfile

```diff
@@ -4,14 +4,15 @@ source 'https://rubygems.org'
 
 ruby File.read('.ruby-version')
 
-gem 'rails', '6.1.3.2'
+gem 'rails', '6.1.4.1'
 
 gem 'pg'
-gem 'puma'
+gem 'puma', '~> 5.4.0'
 
 watt_gem_spec = { git: 'https://github.com/artsy/watt.git', branch: 'master' }
 gem 'watt', watt_gem_spec # artsy bootstrap
 
+gem 'addressable', '>= 2.8.0' # https://github.com/advisories/GHSA-jxhc-q857-3j6g
```

Comment:
I get that we need 2.8.0 in order to have a version without that vulnerability, but don't we get that even without this line? Just not sure why we need to include this gem here rather than relying on `bundle update` to get us on the latest version.

Reply:
No specific reason, I tend to like these more explicit call-outs in the gemfile just for the sake of people reading the file, but it's not necessary.